src/libs/ReportActionsUtils.js

```diff
@@ -179,6 +180,34 @@ function getLastVisibleMessageText(reportID, actionsToMerge = {}) {
         .substring(0, CONST.REPORT.LAST_MESSAGE_TEXT_MAX_LENGTH);
 }
 
+/**
+ * Filter out any reportActions which should not be displayed
+ * Then, sort by their created timestamp first, and reportActionID second
+ * This is a combination of getSortedReportActions() and filterReportActionsForDisplay()
+ *
+ * @param {Object} reportActions
+ * @returns {Array}
+ */
+function getSortedReportActionsForDisplay(reportActions) {
+    // HACK ALERT: We're temporarily filtering out any reportActions keyed by sequenceNumber
+    // to prevent bugs during the migration from sequenceNumber -> reportActionID
+    const filteredReportActions = _.filter(reportActions, (reportAction, key) => {
+        if (!reportAction) {
+            return false;
+        }
+
+        if (String(reportAction.sequenceNumber) === key) {
+            Log.info('Front-end filtered out reportAction keyed by sequenceNumber!', false, reportAction);
+            return false;
+        }
+
+        return true;
+    });
+
+    const sortedReportActions = getSortedReportActions(filteredReportActions, true);
+    return filterReportActionsForDisplay(sortedReportActions);
```

Comment:
Since this is the only reference to `filterReportActionsForDisplay`, I guess it doesn't need it's own method anymore and the logic can be moved inside of this method. Would you agree?

Reply:
`filterReportActionsForDisplay` is being used in https://github.com/Expensify/App/blob/6f2705807eadea43972bc3fdef69237c8c24277a/tests/unit/ReportActionsUtilsTest.js#L141 so I think we shouldn't remove it's own method.

Reply:
That's a great idea from @tgolen! I'm inclined to have `filterReportActionsForDisplay` removed from export and the unit test. Then we can add the `getSortedReportActionsForDisplay` for the unit test. @tienifr Are you okay with that?

Reply:
@mollfpr @tgolen I just removed the UTs of filterReportActionsForDisplay function and add UTs for getSortedReportActionsForDisplay function with the same case in UTs of filterReportActionsForDisplay function because we had UTs for getSortedReportActions function before and this hack logic in the below image will be removed in future. So we just need to add the UTs like in the previous filterReportActionsForDisplay function

<img width="640" alt="Screenshot 2023-02-22 at 23 00 24" src="https://user-images.githubusercontent.com/113963320/220683850-0cb90bda-6973-4deb-8877-7189418e4f5c.png">